universal-application-tool-0.0.1/pom.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
issue (blocking): keeping the pom file up to date is going to be a bit of a hassle since dependency updates are done automatically with renovate bot. instead of checking it in and manually updating it, can you add a script to generate it and gitignore the generated file?

Reply:
Done.  Added a bin/vscode-setup script